core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -1710,6 +1710,9 @@ public function addAliasButton($arrButtons, DataContainer $dc)
 
 				// Create a new version
 				$objVersions->create();
+
+				// Update the record stored in the page registry (see #6542)
+				PageModel::findByPk($id)->alias = $strAlias;
```

Comment:
Should we may be use something like

```php
if ($pageInRegistry = Registry::getInstance()->fetch(PageModel::getTable(), $id))
{
    $pageInRegistry->alias = $strAlias
}
```

instead here? So that it's more clear through the API.

Reply:
It would more tightly bind this to the internals of the registry, I'm unsure whether that's good or bad 🤔